src/types.ts

```diff
@@ -13,6 +13,9 @@ export interface PitcherManagerResponse {
   latestPitcherVersion: string;
   pitcherToken: string;
   cluster: string;
+  vmAgentType?: string;
```

Comment:
this field is not optional, pitcher will always return value

Reply:
There is a build error. I will fix that now